packages/manager/src/features/CloudPulse/shared/CloudPulseDashboardSelect.tsx

```diff
@@ -90,11 +91,12 @@ export const CloudPulseDashboardSelect = React.memo(
         }}
         renderGroup={(params) => (
           <Box key={params.key}>
-            <Typography
-              sx={{ marginLeft: '3.5%', textTransform: 'capitalize' }}
-              variant="h3"
-            >
-              {params.group}
+            <Typography sx={{ marginLeft: '3.5%' }} variant="h3">
+              {
+                serviceTypesList?.data.find(
+                  (type: ServiceTypes) => type.service_type === params.group
+                )?.label
```

Comment:
can we transform the serviceTypesList into map of serviceType to its label, that'll be easy to retrieve the label instead of everytime searching from list

Reply:
added mapping, removed searching